decidim-core/config/locales/en.yml

```diff
@@ -615,8 +615,16 @@ en:
       endorsed_by: Liked by…
     errors:
       internal_server_error:
+        copied: Text copied!
+        copy_error_message_clarification: Copy error message to clipboard
+        copy_to_clipboard: Copy to clipboard
+        date_and_time: 'Date and Time:'
```

Comment:
After we have formatted the information to a table, please remove the semicolons from these strings (all of the information "headers").